### DISCUSSION TITLE
Dịch callback, profiling trace

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#100] [Terminology] [answered]
Từ `callback` chắc nên giữ nguyên, còn từ `profiling trace` không biết nên dịch thế nào nhỉ mọi người?

Context:
* callback trong Keras
* profiling trace
>  There's one directory per run, each containing one subdirectory for training logs and one for validation logs. 
Both contain event files, but the training logs also include **profiling traces**: this allows TensorBoard to show you 
exactly how much time the model spent on each part of your model, across all your devices, 
which is great for locating performance bottlenecks.

cc: @mlbvn/handson-ml

**Comment by rootonchair** [comment]
Em nghĩ nên giữ nguyên 2 từ này: từ profiling trace hay profiler em thấy bên lập trình họ giữ nguyên. Còn nếu dịch thì chắc nên dịch theo ý nghĩa, chẳng hạn như: bản ghi thời gian chạy?

**Comment by cuongvng** [comment]
profiler: công tơ hiệu suất?
profiling trace: bản ghi hiệu suất.

**Comment by minhduc0711** [comment, marked as answer]
Tạm chốt nhé:
* `callback`: giữ nguyên
* `profiling trace`: bản ghi hiệu suất
